cmd/registry/cmd/compute/conformance_test.go

```diff
@@ -33,62 +33,9 @@ import (
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/testing/protocmp"
+	"github.com/apigee/registry/cmd/registry/conformance"
 )
 
-func initReport(t *testing.T) *rpc.ConformanceReport {
```

Comment:
Moved these helper functions to `test-helpers.go`

Reply:
I started with the normal struct approach, constructing each testcase got super long since we had to initialize all the `status` and `severity` values. This was my attempt in avoiding that repeated initialization code and maintain only what is required in the testcases. 

However, I realize now that this approach is making it more complicated and affecting the readability of the tests. I can revert to my original method.

Reply:
Not valid anymore